src/jdk.jfr/share/classes/jdk/jfr/internal/tool/PrettyWriter.java

```diff
@@ -437,13 +437,16 @@ void printObject(RecordedObject object, long arraySize) {
     private void printClassLoader(RecordedClassLoader cl, String postFix) {
         // Purposely not printing class loader name to avoid cluttered output
         RecordedClass clazz = cl.getType();
-        print(clazz == null ? "null" : clazz.getName());
         if (clazz != null) {
+            print(clazz.getName());
             print(" (");
             print("id = ");
             print(String.valueOf(cl.getId()));
-            println(")");
+            print(")");
+        }else{
```

Comment:
please add space after }

Reply:
oh sorry about that, I forgot it this time - old habit. Will fix it.

Reply:
fixed